README.md

```diff
@@ -101,6 +101,18 @@ The key's randomart image is:
 +----[SHA256]-----+
 ```
 
+## VPC
+
+Create a new VPC using the `awscli` command: 
+
+```
+aws ec2 create-vpc --cidr-block 10.0.0.0/8
+```
+
+Make note of the `VpcId` and export it as an environmental variable. 
```

Comment:
I used `jq` to make these instructions even more rote / foolproof. See for example [this line](https://github.com/edgi-govdata-archiving/web-monitoring-ops/pull/21/files#diff-04c6e90faac2675aa89e2176d2eec7d8R336). Would that approach would here too?

Reply:
If I remember correctly, when I ran the commands in the documentation that piped into `jq` they failed. Since I knew what I was looking for I just redid the command and pulled out the variables manually. It's only foolproof if it works :).